server/tabby_server/services/resource_format.py

```diff
@@ -85,11 +85,11 @@ def google_books_test_call_api():
 # Generic function for (hopefully) cleaner readability when it comes to
 # retrieving info about books.
 
-def get_book_attributes(book_list, book_attr):
+def get_book_attribute(book_list, book_attr):
     try:
         return book_list["volumeInfo"][book_attr]
     except KeyError:
-        return {"NA": "Couldn't find!"}
+        return "None"
```

Comment:
It should be the literal `None` instead of the string `"None"`. This corresponds to `null` in the JSON.

Reply:
Other than that, this looks fine.

Reply:
Didn't know that. I've now applied it.